### PR TITLE
Completed desktop version of portfolio

### PR DESCRIPTION
**Pull request by ometman**
The desktop version of the portfolio has been completed.
The index.html, styles.css and stylesdesk.css were refactored.
The hamburger menu was replaced with three main menu items.
Linters were checked and passed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -188,25 +194,173 @@ <h2 class="pr-title">
...
                         have a project, don&#39;t hestiate to contact me.
                     </p> 
                     <!--social media -->
-                    <p class="social-media-text">
+                    <p class="social-media-text abt-social-media-text">
                         LET&#39;S CONNECT 
                     </p>
                     <!--social media icons-->
-                    <ul class="social-media-icons">
+                    <ul class="social-media-icons abt-social-media-icons">
```
Comment (iambenkis) [inline index.html:363]: - [ ] Good job making everything done 👍 , kindly make sure you correct the text in this button is well aligned 
Capture 2

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @ometman  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle:  

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ometman) [reply index.html:363]: Thanks. I will check it. It looked fine locally though.

Comment (ometman) [reply index.html:363]: It was difficult to see the error, but has refactored the styles to effect any such correction. Thanks

File: css/stylesdesk.css
Comment (AbelG101) [inline css/stylesdesk.css:1]: https://github.com/ometman/My-Portfolio/blob/e9b8ae37f8d7f08e4b2007fe3ebf5810c3fb5d77/css/stylesdesk.css#L708-L714
https://github.com/ometman/My-Portfolio/blob/e9b8ae37f8d7f08e4b2007fe3ebf5810c3fb5d77/css/stylesdesk.css#L724-L731
- [ ] [OPTIONAL] I believe it was an oversight on part of the previous reviewer and I'd like to apologize. I really appreciate the effort you put Into placing the `:hover` and `:active` state properties in changing the background color but please make sure they're aligned with the given Figma design.
    - Figma design:
      firefox_7KIb3eS8J1
    - So when hovered over the button the background should change to: `#6070FF` and a subtle box shadow (you can check the design)
    - And when pressing the button the background should change to: `#2230D2`

**Review by AbelG101 — APPROVED** [review]
# Hi @ometman 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- UI is looking on point ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/stylesdesk.css
Comment (ometman) [reply css/stylesdesk.css:1]: Great review! I appreciate the reminders too. I will fix them asap. Thanks.

**Pull request merged**